src/stable/f5-bigip-ctlr/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Deploy the F5 Networks k8s-bigip-ctlr for using the BIG-IP in a Kubernetes or OpenShift environment
```

Comment:
same as above - Deploy the F5 Networks BIG-IP Controller for Kubernetes and OpenShift (k8s-bigip-ctlr)

Reply:
Done.